mediapicker/build.gradle

```diff
@@ -37,27 +34,27 @@ android {
     dependencies {
         api mediaPickerDomainDependency
 
-        implementation "com.google.android.material:material:$googleMaterialVersion"
+        api "com.google.android.material:material:$googleMaterialVersion"
```

Comment:
hi @0nko 👋 what's the reason for using `api` instead of `implementation`? This will pass all dependencies to the consumers of `mediapicker`. I'm not sure if it's intended for libraries.

Reply:
Oh 🤦 I mixed it up, I meant to do the opposite and prevent the internal dependency leak. I'll reverse it.

Reply:
gotcha, thanks! 🙇